SLURM_README.md

```diff
@@ -0,0 +1,55 @@
+# Running Conformance Tests on SLURM Clusters
+## Singularity Cache
+Note: For Singularity, Singularity Cache directories may need to be specified:\
+[toil-wdl-runner](https://giwiki.gi.ucsc.edu/index.php?title=Slurm_Tips_for_Toil)\
+[Cromwell](https://cromwell.readthedocs.io/en/stable/tutorials/Containers/#singularity-cache)\
+[MiniWDL](https://github.com/chanzuckerberg/miniwdl/blob/6dfe83781f74a8e248727eb61c31e1c7562bd26e/tests/singularity.t#L29)
+
+Docker should handle cache by itself.
+## Under Cromwell
+Running on SLURM (and other clusters) may require more arguments/options to be passed into the runners.
+
+A [configuration file](https://cromwell.readthedocs.io/en/stable/Configuring/) must be used to tell Cromwell how to schedule workers under SLURM.
+An example configuration file is in [cromwell-config.conf](examples/cromwell-config.conf). This file specifies to Cromwell how to run in SLURM with Singularity.
+
+The configuration file is then passed into the runner through `--cromwell-pre-args` (not `--cromwell-args` as this must be passed into Java, not Cromwell).
+```commandline
+python run.py --runner cromwell --id tut01 --cromwell-pre-args="-Dconfig.file=examples/cromwell-config.conf"
+```
+
+For SLURM+Docker, the [configuration file](examples/cromwell-config.conf) includes an example configuration, but this has not been fully tested. `backend.default` can be replaced with `SLURM` instead.
+
+## Under MiniWDL
+The [MiniWDL SLURM plugin](https://github.com/miniwdl-ext/miniwdl-slurm) can be used to run MiniWDL in SLURM with Singularity.
+
+The plugin can be [installed](https://github.com/miniwdl-ext/miniwdl-slurm?tab=readme-ov-file#installation) like any other MiniWDL plugin.
+
+MiniWDL then needs to be [configured](https://github.com/miniwdl-ext/miniwdl-slurm?tab=readme-ov-file#configuration) to use this extension. This can be done through a [configuration file or through environmental variables](https://miniwdl.readthedocs.io/en/latest/runner_reference.html?highlight=config#configuration).
+An example configuration file is in [miniwdl-config.cfg](examples/miniwdl-config.cfg). 
+
+No arguments are needed to be passed into MiniWDL as long as MiniWDL is able to read the new configuration.
+An example if MiniWDL specific arguments must be used:
```

Comment:
This should maybe say that it is for the case where you don't want to modify the normal MiniWDL configuration and you need to point it at the customized configuration via argument? We just got finished saying extra arguments aren't needed and now we're talking about the case where actually they "must be used", which is confusing.